database/factories/AssessmentFactory.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/** @var \Illuminate\Database\Eloquent\Factory $factory */
+
+use App\Model;
+use Faker\Generator as Faker;
+use App\Asssessment;
+
+$factory->define(Assessment::class, function (Faker $faker) {
+    return [
+        'section_id' => 1,
```

Comment:
when referencing a foreign key, you should do this instead:

```
'section_id' => function () {
    return factory(Section::class)->create()->id;
}
```
This way, the relation will really exist, which is useful for testing.

When using the factory, if you override the `section_id`, it will just use the supplied value and not create the model specified in the factory definition.

eg.

```
factory(Assessment::class)->create(['section_id' => factory(Section::class)->create()->id]);
```